src/NServiceBus.Unity.AcceptanceTests/NServiceBus.Unity.AcceptanceTests.csproj

```diff
@@ -12,8 +12,8 @@
   <ItemGroup>
     <PackageReference Include="NServiceBus" Version="7.0.0-*" />
     <PackageReference Include="NServiceBus.AcceptanceTesting" Version="7.0.0-*" />
-    <PackageReference Include="Unity" Version="4.*" />
-    <PackageReference Include="Unity.Interception" Version="4.*" />
```

Comment:
might be worth adding a comment as to why here. i remember i hit it as well

Reply:
Good idea